src/CoreShop/Bundle/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -151,9 +151,11 @@ private function addPimcoreResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('payment')->defaultValue('/bundles/coreshoppayment/pimcore/css/payment.css')->end()
                         ->end()
                     ->end()
-                    ->scalarNode('permissions')
-                        ->cannotBeOverwritten()
-                        ->defaultValue(['payment_provider'])
+                    ->arrayNode('permissions')
+                        ->scalarPrototype()
```

Comment:
don't overwrite permission configurations, you cannot change them anyway and its not part of this PR